extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcClientProcessor.java

```diff
@@ -277,6 +280,39 @@ public void validateInjectedServiceInterfaces(CombinedIndexBuildItem index,
         }
     }
 
+    @BuildStep
+    InjectionPointTransformerBuildItem transformInjectionPoints() {
+        return new InjectionPointTransformerBuildItem(new InjectionPointsTransformer() {
+
+            @Override
+            public void transform(TransformationContext ctx) {
+                // If annotated with @GrcClient and no explicit value is used, i.e. @GrcClient(), 
```

Comment:
I think "grc" is some Moravian dialect, but otherwise it looks fine :laughing:

Reply:
Oh my...